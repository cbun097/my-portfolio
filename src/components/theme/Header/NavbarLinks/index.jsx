import React from 'react'
import AnchorLink from 'react-anchor-link-smooth-scroll'
import { Wrapper } from './styles'

const NavbarLinks = ({ desktop }) => (
	<Wrapper desktop={desktop}>
		<AnchorLink href="#about">About</AnchorLink>
		<AnchorLink href="#projects">Projects</AnchorLink>
		<AnchorLink href="#contact">Contact</AnchorLink>
		<AnchorLink href="#blog">Blog</AnchorLink>
	</Wrapper>
)

export default NavbarLinks
