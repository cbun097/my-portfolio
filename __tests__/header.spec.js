import React from 'react'
import renderer from 'react-test-renderer'
import { Header } from '../src/components/theme/Header/index'

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
