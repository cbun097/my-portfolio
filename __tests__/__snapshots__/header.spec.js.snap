// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="sc-fznZeY cgAvji"
>
  <div
    className="sc-fzokOt AUdI"
    onClick={[Function]}
  />
  <div
    className="sc-AxhUy eJhSOM sc-fzpans iRvtGb"
  >
    <a
      aria-current="page"
      className=""
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      style={Object {}}
    >
      Claire Bun
    </a>
    <div
      className="sc-fzoLsD hLUwpL"
    >
      <a
        href="#about"
        onClick={[Function]}
      >
        About
      </a>
      <a
        href="#projects"
        onClick={[Function]}
      >
        Projects
      </a>
      <a
        href="#blog"
        onClick={[Function]}
      >
        Blog
      </a>
      <a
        href="#contact"
        onClick={[Function]}
      >
        Contact
      </a>
    </div>
  </div>
  <div
    className="sc-fzplWN ggXYHH"
    onClick={[Function]}
  >
    <div
      className="sc-fznyAO cVabKd"
    />
    <div
      className="sc-fznyAO cVabKd"
    />
    <div
      className="sc-fznyAO cVabKd"
    />
  </div>
  <div
    className="sc-fznKkj dJlsUJ"
    onClick={[Function]}
  >
    <div
      className="sc-fzoLsD kYqwTH"
    >
      <a
        href="#about"
        onClick={[Function]}
      >
        About
      </a>
      <a
        href="#projects"
        onClick={[Function]}
      >
        Projects
      </a>
      <a
        href="#blog"
        onClick={[Function]}
      >
        Blog
      </a>
      <a
        href="#contact"
        onClick={[Function]}
      >
        Contact
      </a>
    </div>
  </div>
</div>
`;
